src/typeDefs/mutations.ts

```diff
@@ -9,6 +9,8 @@ export const mutations = gql`
 
```

Comment:
Use a fixed approach for writing schema definitions. This will create uniformity across the schema and make it easier to traverse through it manually. Graphql has no convention or strict rules for schema design, it is the developer's job to make conventions and strict rules for writing graphql schema for their applications. Here I'm listing some good conventions/patterns for mutations in graphql schema design:-

1. First you need name of the types/objects the mutation is to be implemented on. Then create mutation names by prefixing the name of those types/objects with action words like create, add, update, delete, remove etc. 

2. Have each mutation accept a single argument named input. Since the input is abstracted away into its own type in the schema it is much more cleaner and much more flexible to extend if needed later on. Make the input for each mutation required, it doesn't make sense to trigger a mutation without the input object. In exceptional cases input might not be needed.

3. If needed create a custom mutation result type to communicate to the client result and errors in a proper, easy to understand format.

Fixing the already existing resolvers is not as feasible because they've already been implemented on the client side and you'd need to update client side code as well. But the new ones you're created here can be written from the start to follow a good convention.

Reply:
I have tried to follow the same as far as possible. In all of the mutations I have added, I have made sure only to use a singular input type for all the inputs as far as possible. The only time I have used two inputs, are when a `id` and some `data` input are required for the same, as it makes sense to take them separately IMO as the `id` is used in resolver in multiple places for auth checks and fetching, while the data is usually destructed in the resolver to populate. 

If you find any resolvers that differ from this convention, you can let me know and I will fix them.